packages/mds-activity-service/logger.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2022 City of Los Angeles
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createLogger } from '@mds-core/mds-logger'
+
+export const ActivityServiceLogger = createLogger('mds-activity-service')
+
+export const ActivityLogger = createLogger('mds-activity')
```

Comment:
Do we need both? 🤔

Reply:
Maybe, maybe not. I was thinking the latter would just echo every activity that was recorded while the former would be used more for debugging the service itself, but they _could_ be the same thing.

Reply:
I consolidated.